lib/sa.ts

```diff
@@ -49,7 +49,7 @@ export const createServiceAccount = async function (page: Page, name: string) {
 };
 
 export const deleteServiceAccount = async function (page: Page, name: string) {
-  const saLinkSelector = page.getByText(name, { exact: true });
+  const saLinkSelector = page.getByText(name);
```

Comment:
What was wrong with `exact`?

Reply:
![image](https://user-images.githubusercontent.com/100594859/216299729-3c253aa7-5da6-4ba3-a3b9-fe3350de5a33.png)

For some reason, the exact is rigid that on that method that `saLinkSelector.count()` was evaluating to "0" avoiding to delete Service Account. I was able to see that on debugging

Reply:
maybe use something better than `getByText` would make it more robust as well

Reply:
replaced for `const saLinkSelector = page.locator('td', { hasText: `${name}` });`

Reply:
Can I resolve this conversation @Frawless @mmusil ?